### DISCUSSION TITLE
prevent white space before any character in input filled

### DISCUSSION THREAD

**Discussion by Tihitna-22** [mantinedev/mantine#4400] [Q&A]
Is here any built-in props that will prevent white space before inserting any character so that it will not consider white space as a character in Textinput and Textarea
  
              <TextInput
              className="mb-2"
              placeholder="Please enter Title"
              label="Title"
              error={error}
              error={
              errors?.['name'] ? errors?.['name']?.message : ''
              }
              required
              {...register('name')}
              />
                <Textarea
                className="mb-2"
                required
                placeholder="Please enter Description"
                label="Description"
                error={
                errors?.['description'] ? errors?.['description']?.message : ''
                }
                {...register('description')}
                />

              where i use yup for validation 
              const {
              register,
              handleSubmit,
              formState: { errors },
              reset,
              setValue,
              control,
              } = useForm({ resolver: yupResolver(schema) });

**Comment by rtivital** [comment]
Use controlled input state, call `.trimStart()` on the value
